src/app/App.js

```diff
@@ -200,6 +209,19 @@ export default function App() {
           />
         )}
       />
+      <Route
+        exact
+        path="/favPlants"
+        render={props => (
+          <FavPage
+            plants={getFavSortedFilteredPlants()}
+            onBookmark={handleBookmark}
+            {...props}
```

Comment:
Right now it looks like you are not using the props. If that's the case, you can delete them.